addons/languages/coptic/pack/src/main/res/xml/coptic_scriptorium.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <Keyboard xmlns:android="http://schemas.android.com/apk/res/android"
-    android:keyWidth="10%p"><!-- "android:keyWidth" specify the default width of a key. In this example, 10% of parent (the whole keyboard) -->
+    android:keyWidth="9.09%p"><!-- "android:keyWidth" specify the default width of a key. In this example, 10% of parent (the whole keyboard) -->
```

Comment:
Not sure about changing the key width, I already have trouble getting the correct key. I'll give it a try though.

Anyway, the comment should be updated to match, maybe just to say that 10% is the usual width.